ush/templates/FV3LAM_wflow.xml

```diff
@@ -263,9 +279,9 @@ MODULES_RUN_TASK_FP script.
     <dependency>
       <!-- Files through fhr 39 exist. -->
       <sh>
-        test $(find <cyclestr offset="-{{ extrn_mdl_lbcs_offset_hrs }}:00:00">{{ extrn_mdl_sysbasedir_lbcs }}/@Y@m@d@H/postprd</cyclestr> \
-               -name wrfnat_rr_[0-3][0-9].grib2 -mmin +5 | wc -l) \
-        == 40
```

Comment:
We do need all the LBCs available for the full forecast time here.

Reply:
We only need 18 hours for the data assimilation. Those checks should be set based on the forecast length in the future. Or juts check the last forecast needed for the boundary conditions.

Reply:
This part should be controlled by the configuration.

Reply:
Can you please add a template section for that, then? This will surely get missed when we update the forecast length, otherwise.

Reply:
I think to check the longest forecast result file (single file) is good enough here.
I reverted to original value. Will change this check in later PR.
Also, we will need prepare much longer boundary condition than the forecast length when use GFS.